Makefile

```diff
@@ -50,7 +50,7 @@ test-e2e:
 	./test/e2e-tests.sh
 
 # Run E2E tests from the current repo for serving+eventing+knativeKafka
-test-e2e-with-kafka:
+test-e2e-with-kafka: test-ui-e2e
```

Comment:
This is temporal change, right?

/hold

Reply:
Right. No need to hold. This PR is WIP